analysis/measures_definition.py

```diff
@@ -73,13 +73,16 @@
 
 # Demographic variables and other patient characteristics
 # Define variable that checks if a patients is registered at the start date
-has_registration = practice_registrations.for_patient_on(
```

Comment:
`has_registration` is common variable name and I think we should stick with it unless there's a good reason not to.